[2022-06-11 18:56:18,998] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: T-dbt-job-cdc_load.dbt_cleanup-cdc_load manual__2022-06-11T18:56:15.146767+00:00 [queued]>
[2022-06-11 18:56:19,009] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: T-dbt-job-cdc_load.dbt_cleanup-cdc_load manual__2022-06-11T18:56:15.146767+00:00 [queued]>
[2022-06-11 18:56:19,009] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-11 18:56:19,009] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-11 18:56:19,009] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-11 18:56:19,020] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): dbt_cleanup-cdc_load> on 2022-06-11 18:56:15.146767+00:00
[2022-06-11 18:56:19,026] {standard_task_runner.py:52} INFO - Started process 1784 to run task
[2022-06-11 18:56:19,030] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'T-dbt-job-cdc_load', 'dbt_cleanup-cdc_load', 'manual__2022-06-11T18:56:15.146767+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/docker_job/dbt_test.py', '--cfg-path', '/tmp/tmph645s5_p', '--error-file', '/tmp/tmpc6dsl4fs']
[2022-06-11 18:56:19,030] {standard_task_runner.py:80} INFO - Job 109: Subtask dbt_cleanup-cdc_load
[2022-06-11 18:56:19,080] {task_command.py:370} INFO - Running <TaskInstance: T-dbt-job-cdc_load.dbt_cleanup-cdc_load manual__2022-06-11T18:56:15.146767+00:00 [running]> on host d8836eb746ac
[2022-06-11 18:56:19,146] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=samuelolle@yahoo.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=T-dbt-job-cdc_load
AIRFLOW_CTX_TASK_ID=dbt_cleanup-cdc_load
AIRFLOW_CTX_EXECUTION_DATE=2022-06-11T18:56:15.146767+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-11T18:56:15.146767+00:00
[2022-06-11 18:56:19,147] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-11 18:56:19,148] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm -rf /tmp/cdc_load && rm -rf /tmp/tmp* 2>/dev/null || exit 0']
[2022-06-11 18:56:19,160] {subprocess.py:85} INFO - Output:
[2022-06-11 18:56:19,166] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-11 18:56:19,193] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=T-dbt-job-cdc_load, task_id=dbt_cleanup-cdc_load, execution_date=20220611T185615, start_date=20220611T185618, end_date=20220611T185619
[2022-06-11 18:56:19,242] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-11 18:56:19,271] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
